sql-odbc/src/odfesqlodbc/es_parse_result.cpp

```diff
@@ -29,6 +29,7 @@
 #pragma clang diagnostic pop
 #endif  // __APPLE__
 #include "statement.h"
+#include "es_types.h"
```

Comment:
Already included on L22 (might need to expand GH code preview)

Reply:
removed